meta/sai_meta_schedulergroup.cpp

```diff
@@ -23,7 +23,7 @@ const sai_attr_metadata_t sai_scheduler_group_attr_metadata[] = {
         .attrid                 = SAI_SCHEDULER_GROUP_ATTR_CHILD_LIST,
         .serializationtype      = SAI_SERIALIZATION_TYPE_OBJECT_LIST,
         .flags                  = SAI_ATTR_FLAGS_READ_ONLY,
-        .allowedobjecttypes     = { SAI_OBJECT_TYPE_SCHEDULER }, // TODO there may be different types like QUEUE or PORT here
+        .allowedobjecttypes     = { SAI_OBJECT_TYPE_SCHEDULER, SAI_OBJECT_TYPE_SCHEDULER_GROUP }, // TODO there may be different types like QUEUE or PORT here
```

Comment:
I dont get the comment, can you list all valid object types that can be on this list ?